include/itkPhaseCorrelationImageRegistrationMethod.h

```diff
@@ -227,7 +227,7 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationImageRegistrationMethod: public Proces
   /** \class PaddingMethod
    *  \brief Different methods of padding the images to satisfy FFT size requirements.
    *  \ingroup Montage */
-  enum class PaddingMethod { Zero, Mirror, MirrorWithExponentialDecay };
+  enum class PaddingMethod :short { Zero = 0, Mirror, MirrorWithExponentialDecay, Last = MirrorWithExponentialDecay };
```

Comment:
To keep things simple for wrapping, etc, we should keep the default integral type.